rbac/management/role/serializer.py

```diff
@@ -169,7 +164,7 @@ def update(self, instance, validated_data):
         for access_item in access_list:
             resource_def_list = access_item.pop("resourceDefinitions")
             access_permission = access_item.pop("permission")
-            permission, created = Permission.objects.get_or_create(**access_permission)
+            permission, created = Permission.objects.get_or_create(**access_permission, tenant=tenant)
```

Comment:
Do we need to be concerned with creating duplicates here prior to the tenant_id sync job being run without something like this? https://github.com/RedHatInsights/insights-rbac/pull/467/files#diff-ac43ba775b89d5e57f70fdea347fe345ab148950cf8650dd14f1af761edd07e2R139-R149

Reply:
That's a good point, and there likely is a risk here. I could try to write up a version where we do a get, then check if we got anything, and if not then we create with the tenant. It'd be easiest if we could push up the sync command first, but it won't work without these in place, I fear.